base/base.bzl

```diff
@@ -8,47 +8,68 @@ NONROOT = 65532
+                },
+                tars = [
+                    ":passwd",
+                    ":group_tar",
+
+                    # Create /tmp, too many things assume it exists.
+                    # tmp.tar has a /tmp with the correct permissions 01777
+                    # A tar is needed because at the moment there is no way to create a
+                    # directory with specific permissions.
+                    ":tmp.tar",
+                    ":nsswitch.tar",
+                    DISTRO_REPOSITORY[distro_suffix] + "//file:os_release.tar",
+                    ":cacerts_" + arch + distro_suffix + ".tar",
+                ],
+                user = "%d" % uid,
+                workdir = workdir,
+            )
+
+    for (user, uid, workdir) in [("root", 0, "/"), ("nonroot", NONROOT, "/home/nonroot")]:
+        native.alias(
+            name = "static_" + user + distro_suffix,
+            actual = ":static_" + user + "_amd64" + distro_suffix,
+        )
+
+    native.alias(
         name = "cacerts" + distro_suffix,
-        deb = DISTRO_PACKAGES[distro_suffix]["ca-certificates"],
+        actual = ":cacerts_amd64" + distro_suffix,
     )
 
-    container_image(
+    native.alias(
         name = "static" + distro_suffix,
-        debs = [
-            DISTRO_PACKAGES[distro_suffix]["base-files"],
-            DISTRO_PACKAGES[distro_suffix]["netbase"],
-            DISTRO_PACKAGES[distro_suffix]["tzdata"],
-        ],
-        env = {
-            "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-            # allows openssl to find the certificates by default
-            # TODO: We should run update-ca-certifaces, but that requires "openssl rehash"
-            # which would probably need to be run inside the container
-            "SSL_CERT_FILE": "/etc/ssl/certs/ca-certificates.crt",
-        },
-        tars = [
-            ":passwd",
-            ":group_tar",
-
-            # Create /tmp, too many things assume it exists.
-            # tmp.tar has a /tmp with the correct permissions 01777
-            # A tar is needed because at the moment there is no way to create a
-            # directory with specific permissions.
-            ":tmp.tar",
-            ":nsswitch.tar",
-            DISTRO_REPOSITORY[distro_suffix] + "//file:os_release.tar",
-            ":cacerts" + distro_suffix + ".tar",
-        ],
+        actual = ":static_root_amd64" + distro_suffix,
     )
 
     container_image(
         name = "base" + distro_suffix,
         base = ":static" + distro_suffix,
         debs = [
-            DISTRO_PACKAGES[distro_suffix]["libc6"],
-            DISTRO_PACKAGES[distro_suffix]["libssl1.1"],
-            DISTRO_PACKAGES[distro_suffix]["openssl"],
+            DISTRO_PACKAGES["amd64"][distro_suffix]["libc6"],
+            DISTRO_PACKAGES["amd64"][distro_suffix]["libssl1.1"],
+            DISTRO_PACKAGES["amd64"][distro_suffix]["openssl"],
```

Comment:
cc @dekkagaijin 

I hear you want a distroless/base (with glibc) for use with K8s stuff?  In this PR I'm faking `arm64` support, but if we can plumb through reading the package lists, then we can extend my "arch" refactor to include at least this, and probably some of the java and python targets.

If we push through a second architecture all the way, I can volunteer to test ARM64 on a mix of my RPi4 and a Graviton2 EKS cluster.